UnraisinYourself/UI Extensions/View+AutoLayout.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  View+AutoLayout.swift
+//  UnraisinYourself
+//
+//  Created by Hannie Kim on 10/9/23.
+//
+
+import Foundation
+import UIKit
+
+extension UIView {
+    func anchor(
```

Comment:
Helper for adding constraints to a view. It'll automatically set translatesAutoresizingMaskIntoConstraints to false